### PR TITLE
Validate input type - fixes #1 

### LINKED ISSUE #1: Check user input
It might be an idea to check the type of the input and throw a `TypeError` when it's not a `Date` or a `number`.

I can do a PR if you want.

### PR DESCRIPTION
**Pull request by SamVerschueren**
Check the input type and throw a `TypeError` if it's not a `Date` or a `number`.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -2,6 +2,7 @@ import test from 'ava';
 import m from './';
 
 test(t => {
+	t.throws(() => m('foo'), 'Expected `year` to be of type `Date` or `number`, got `string`');
```
Comment (sindresorhus) [inline test.js]: Can you just check for a TypeError here? I don't like having to duplicate the error message in multiple places.

**Comment by sindresorhus** [comment]
screen shot 2016-12-17 at 21 46 37

Lol :p ?

**Comment by SamVerschueren** [comment]
Hahaha, on mobile. Missed that you changed the title first :laughing:, sorry.

**Comment by sindresorhus** [comment]
No worries. I'm trying out uppercasing the first character in PRs and commits. Kinda looks nicer and more like a sentence.

**Comment by SamVerschueren** [comment]
That's why I thought I screwed up the PR, because of the uppercase. Good to know :).

**Pull request merged**